cfgov/paying_for_college/documents.py

```diff
@@ -0,0 +1,60 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+from elasticsearch_dsl import analyzer, token_filter, tokenizer
+
+from paying_for_college.models import School
+
+
+label_autocomplete = analyzer(
+    'label_autocomplete',
+    tokenizer=tokenizer(
+        'trigram',
+        'edge_ngram',
+        min_gram=2,
+        max_gram=25,
+        token_chars=["letter", "digit"]
+    ),
+    filter=['lowercase', token_filter('ascii_fold', 'asciifolding')]
+)
+
+synonynm_filter = token_filter(
+    'synonym_filter_en',
+    'synonym',
+    synonyms_path='/usr/share/elasticsearch/config/synonyms/synonyms_en.txt'
+)
+
+
+@registry.register_document
+class SchoolDocument(Document):
+
+    autocomplete = fields.TextField(analyzer=label_autocomplete)
+    text = fields.TextField(attr='settlement_school')
+    nicknames = fields.TextField()
+    city = fields.TextField()
+    state = fields.TextField()
+
+    def prepare_autocomplete(self, instance):
+        alias_strings = [a.alias for a in instance.alias_set.all()]
+        nickname_strings = [n.nickname for n in instance.nickname_set.all()]
+        return alias_strings + nickname_strings
+
+    def prepare_city(self, instance):
+        return instance.city
```

Comment:
We can use a fields list instead of a prepare function if we aren't making any changes to a field.

https://django-elasticsearch-dsl.readthedocs.io/en/latest/quickstart.html